### PR TITLE
FCCEUD-26 Added path to documentation in README to make it easy to find.

### PR DESCRIPTION
**Pull request by stoobie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,3 +125,7 @@ chromium-browser --disable-web-security --user-data-dir=/home/user/anywhere/chro
 This works because there is code in app.tsx that preempts all fetch() calls and checks if the app is being served from localhost. If so, it modifies the request to point to localhost:8080 specifically, rather than localhost:9000 which is where webpack-dev-server serves the frontend code from.
 
 It might be possible to improve this technique. Suggestions are welcome.
+
+### End user documentation
+
+The source for the end user documentation is stored in /pantheon-bundle/src/main/resources/SLING-INF/content/docs/ref_prerequisites.adoc.
```
Comment (robin-owen) [inline README.md]: I'd just mark up the path in monospace, other than that, this is great. Thank you! :+1:

**Comment by stoobie** [comment]
So who does the actual merge?

**Comment by benradey** [comment]
@stoobie I'll need to do it, but since we're having more trouble with Travis CI and I've reached out to their support team, I'm going to leave this open until I hear back from them. I also commented the same in the matching Jira ticket. Please be patient, it seems we have a few kinks to work out with our workflow now that we have authors opening PRs!

**Comment by stoobie** [comment]
OK, I will have patience. Ommmmmmmmmmmmmmm.

**Pull request merged**